src/components/flag/flag.jsx

```diff
@@ -1,47 +1,70 @@
 import React, { PropTypes } from 'react';
-import classNames from 'classnames';
-import { kebabCase } from 'lodash';
 import flags from './flags';
 
 function Flag({
   className,
   style,
   countryCode,
-  width,
-  height,
+  secondaryCountryCode,
+  size,
   ...rest
 }) {
-  const flag = flags[countryCode.toLowerCase()];
-  const classes = classNames('flag', `flag--${kebabCase(countryCode)}`, className);
+  const PrimarySvgFlag = flags[countryCode.toLowerCase()];
+  const SecondarySvgFlag = secondaryCountryCode ? flags[secondaryCountryCode.toLowerCase()] : null;
+  // width / height ratio of our SVGs are 4 / 3
+  const width = size * (4 / 3);
+  const height = size;
+  const circleSize = size / 2.1;
+  const canadianLeftPos = -size * (3 / 16);
+
   const styles = Object.assign({
     display: 'inline-block',
-    width: `${width / 10}rem`,
-    height: `${height / 10}rem`,
+    position: 'relative',
+    width: `${size}px`,
```

Comment:
You dont need to append `px` to size here or below, `width: size,` will evaluate to px